www/css/dark.css

```diff
@@ -1,7 +1,8 @@
 body {
-  --bg-color: black;
+  --bg-color: #160f19;
   --txt-color: white;
   --disabled-color: lightgray;
+  --a-color: #0af;
 }
 
```

Comment:
Guilds without a configuration are greyed out in the admin menu.

This does not work on the dark theme, and needs to be updated.

I'll probably do this myself since I can see what I'm working with.

![image](https://user-images.githubusercontent.com/32238504/62171743-a6d49180-b338-11e9-9093-16a46d6746ee.png)

![image](https://user-images.githubusercontent.com/32238504/62171789-dbe0e400-b338-11e9-9fe6-11ef04c1ce73.png)

Reply:
why are they hyperlinked if your just going to gray them out?

Reply:
They are grayed out as an inidcation that their configuration is empty.

This property is currently handled via a direct `style` attribute, so it should be changed to a CSS class.

Reply:
![image](https://user-images.githubusercontent.com/33164598/62177671-031cbe80-b303-11e9-9e14-c8797c9d9899.png)